### PR TITLE
fix typo in readme

### PR DESCRIPTION
**Pull request by b-admike**
### Description

Fix typo in readme and rename model name in middleware example snippet to be the same as the remote method example. Also fix the module name in contributing file.

#### Related issues

- None

### Checklist

- [x] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

**Comment by b-admike** [comment]
@bajtos PTAL

File: README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ module.exports = function(options) {
       return next();
     }
 
-    app.models.UserModel.findById(req.accessToken.userId, function(err, user) {
+    app.models.YourModel.findById(req.accessToken.userId, function(err, user) {
```
Comment (bajtos) [inline README.md]: This is intentionally using a different model name than the remote method below. We want to load the User instance here, while the remote method below can be added to any application model.

Can you please revert this line?

**Review by bajtos — CHANGES_REQUESTED** [review]
Hi @b-admike, thank you for taking time to improve our docs 🙇 

Most of the changes looks good to me, there is one part to discuss though - see my comment below.

@josieusa could you PTAL too please?

**Review by josieusa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (b-admike) [reply README.md]: Okay I misunderstood that, thanks Miroslav.

**Review by bajtos — APPROVED** [review]

**Comment by bajtos** [comment]
Landed, thank you for the contribution!

**Pull request merged**